nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/src/main/java/org/apache/nifi/processors/iceberg/converter/GenericDataConverters.java

```diff
@@ -48,10 +51,28 @@
 public class GenericDataConverters {
 
     static class SameTypeConverter extends DataConverter<Object, Object> {
+        final Type.PrimitiveType targetType;
+        final DataType sourceType;
+
+        public SameTypeConverter(final Type.PrimitiveType type, final DataType dataType) {
+            targetType = type;
+            sourceType = dataType;
```

Comment:
This sourceType variable is never used so it can be removed from the class and the constructor.

Reply:
It's used in the `DataTypeUtils.getDateTimeFormatter()` method